packages/config/config/devices/0x027a/zen72.json

```diff
@@ -0,0 +1,429 @@
+// Zooz ZEN72
+// Z-Wave Plus 700 Series Dimmer Switch
+{
+	"manufacturer": "Zooz",
+	"manufacturerId": "0x027a",
+	"label": "ZEN72",
+	"description": "Z-Wave Plus 700 Series Dimmer Switch",
+	"devices": [
+		{
+			"productType": "0x7000",
+			"productId": "0xa002"
+		}
+	],
+	"firmwareVersion": {
+		"min": "0.0",
+		"max": "255.255"
+	},
+	"associations": {
+		"1": {
+			"label": "Lifeline",
+			"maxNodes": 1,
+			"isLifeline": true
+		},
+		"2": {
+			"label": "On/Off Control",
+			"maxNodes": 5
+		},
+		"3": {
+			"label": "Dimmer Control",
+			"maxNodes": 5
+		}
+	},
+	"paramInformation": {
+		"1": {
+			"label": "Paddle Orientation",
+			"valueSize": 1,
+			"minValue": 0,
+			"maxValue": 2,
+			"defaultValue": 0,
+			"readOnly": false,
+			"writeOnly": false,
+			"allowManualEntry": false,
+			"options": [
+				{
+					"label": "Top on, bottom off",
+					"value": 0
+				},
+				{
+					"label": "Top off, bottom on",
+					"value": 1
+				},
+				{
+					"label": "Toggle",
+					"value": 2
+				}
+			]
+		},
+		"2": {
+			"label": "LED Indicator Control",
+			"valueSize": 1,
+			"minValue": 0,
+			"maxValue": 3,
+			"defaultValue": 0,
+			"readOnly": false,
+			"writeOnly": false,
+			"allowManualEntry": false,
+			"options": [
+				{
+					"label": "LED is on when switch is off",
+					"value": 0
+				},
+				{
+					"label": "LED is on when switch is on",
+					"value": 1
+				},
+				{
+					"label": "LED is always off",
+					"value": 2
+				},
+				{
+					"label": "LED is always on",
+					"value": 3
+				}
+			]
+		},
+		"23": {
+			"label": "LED Indicator Color",
+			"valueSize": 1,
+			"minValue": 0,
+			"maxValue": 3,
+			"defaultValue": 1,
+			"readOnly": false,
+			"writeOnly": false,
+			"allowManualEntry": false,
+			"options": [
+				{
+					"label": "White",
+					"value": 0
+				},
+				{
+					"label": "Blue",
+					"value": 1
+				},
+				{
+					"label": "Green",
+					"value": 2
+				},
+				{
+					"label": "Red",
+					"value": 3
+				}
+			]
+		},
+		"24": {
+			"label": "LED Indicator Brightness",
+			"valueSize": 1,
+			"minValue": 0,
+			"maxValue": 2,
+			"defaultValue": 1,
+			"readOnly": false,
+			"writeOnly": false,
+			"allowManualEntry": false,
+			"options": [
+				{
+					"label": "100%",
+					"value": 0
+				},
+				{
+					"label": "60%",
+					"value": 1
+				},
+				{
+					"label": "30%",
+					"value": 2
+				}
+			]
+		},
+		"3": {
+			"label": "Auto Turn-Off Timer",
+			"description": "Set the number of minutes to automatically turn off switch once it has been turned on (0 to disable).",
+			"valueSize": 4,
+			"minValue": 0,
```

Comment:
Please use the unit to below valueSize throughout. We like to keep these uniform.

Reply:
I don't understand the comment here, can you clarify?  Also, shall I update the Zen77 as well since this is almost an identical copy of the existing Zen77 with simply the addition of the 3-way momentary switch option.

Reply:
Typo. We have a specific order for these fields. The unit should be directly under the valueSize.

Reply:
gotcha, I'll remember that.  Fix coming shortly.